backend/app/services/adapters/task_kinds/operations.py

```
@@ -210,6 +211,19 @@ def _create_new_task(
                 detail="Team not found, it may be deleted or not shared",
             )
 
+        if obj_in.project_id:
+            project_exists = (
+                db.query(Project.id)
+                .filter(
+                    Project.id == obj_in.project_id,
+                    Project.user_id == user.id,
+                    Project.is_active == True,
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `Project.is_active:` for truth checks

Replace with `Project.is_active`

(E712)